src/sscofpmf.adoc

```diff
@@ -62,9 +62,9 @@ bit [61] +++SINH+++ - If set, then counting of events in S/HS-mode is inhibited
 
 bit [60] +++UINH+++ - If set, then counting of events in U-mode is inhibited
 
-bit [59] +++VSINH+++ - If set, then counting of events in VS-mode is inhibited
+bit [59] +++VSINH+++ - If set and the H extension is implemented, then counting of events in VS-mode is inhibited
 
-bit [58] +++VUINH+++ - If set, then counting of events in VU-mode is inhibited
+bit [58] +++VUINH+++ - If set and the H extension is implemented, then counting of events in VU-mode is inhibited
```

Comment:
@gfavor these seem odd.  Shouldn't these bits be read-only zero when the H extension isn't implemented, rendering these two changes unnecessary?  (Perhaps their being read-only zero is what needs to be clarified.)

Reply:
(I double-checked that Spike does it the way I suggested.)

Reply:
You're correct as to the not so appropriate wording.  (This change wording came from an issue raised a long time ago.)

But what does still need to be specified is that bits 58 and 59 are read-only-zero when H isn't implemented.  The existing sentence should remain unchanged, and then a second sentence added that says the preceding.

So how about (using bit 58 as the example):

_If set, then counting of events in VU-mode is inhibited.  But if the H extension is not implemented, then this bit is read-only-zero._

Reply:
SGTM.